github/billing.go

```diff
@@ -87,6 +106,20 @@ func (s *BillingService) GetStorageBillingOrg(ctx context.Context, org string) (
 	return storageOrgBilling, resp, err
 }
 
+// GetAdvancedSecurityActiveCommittersOrg returns the GitHub Advanced Security active committers for an organization per repository.
+//
+// GitHub API docs: https://docs.github.com/en/rest/reference/billing#get-github-advanced-security-active-committers-for-an-organization
+func (s *BillingService) GetAdvancedSecurityActiveCommittersOrg(ctx context.Context, org string) (*ActiveCommitters, *Response, error) {
+	u := fmt.Sprintf("orgs/%v/settings/billing/advanced-security", org)
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+	ActiveOrgCommitters := new(ActiveCommitters)
+	resp, err := s.client.Do(ctx, req, ActiveOrgCommitters)
+	return ActiveOrgCommitters, resp, err
```

Comment:
Second... after every `s.client.Do`, please add this code (including the trailing blank line):

```
if err != nil {
  return nil, resp, err
}

```